src/main/scala/it/pagopa/interop/tenantmanagement/api/impl/TenantApiServiceImpl.scala

```diff
@@ -142,7 +142,7 @@ class TenantApiServiceImpl(
       _          <- commanderForTenantId(tenantId).askWithStatus[Unit](DeleteTenantMail(tenantUuid, mailId, _))
     } yield ()
 
-    onComplete(result) { deleteTenantResponse[Unit](operationLabel)(_ => deleteTenant204) }
+    onComplete(result) { deleteTenantResponse[Unit](operationLabel)(_ => deleteTenantMail204) }
```

Comment:
As per previous comment
```suggestion
    onComplete(result) { deleteTenantMailResponse[Unit](operationLabel)(_ => deleteTenantMail204) }
```